cdc/log.cc

```diff
@@ -215,8 +228,23 @@ namespace cdc {
 using operation_native_type = std::underlying_type_t<operation>;
 using column_op_native_type = std::underlying_type_t<column_op>;
 
+static sstring cdc_log_suffix = "_scylla_cdc_log";
+
+bool is_log_name(const std::string_view& table_name) {
+    return boost::ends_with(table_name, cdc_log_suffix);
+}
+
+bool is_log_for_some_table(const sstring& ks_name, const std::string_view& table_name) {
+    if (!is_log_name(table_name)) {
+        return false;
+    }
+    const auto base_view = table_name.substr(0, table_name.size() - cdc_log_suffix.size());
+    const auto base_name = sstring(base_view.data(), base_view.size());
```

Comment:
Maybe we could create sstring from table_name straing away without creating base_view?

Reply:
Done